pkg/api/params/params.go

```diff
@@ -471,8 +471,8 @@ type PurgeLogsResponse struct {
 // target controller must be specified with both the source model and
 // target controller residing within JIMM.
 type MigrateModelInfo struct {
-	// ModelTag is a tag of the form "model-<UIID>"
-	ModelTag string `json:"model-tag"`
+	// TargetModelNameOrUUID is a tag of the form "model-<UUID>/model-<owner/name>"
```

Comment:
It's no longer a tag right? So there is no "model-" prefix.

Reply:
Yup, updated